js/data/question.jsonc

```diff
@@ -0,0 +1,599 @@
+	"steel": "鋼材",
+	"baux": "ボーキ",
+	"bucket": "バケツ",
+	"repair_team": "ダメコン",
+	"avg_repair": "平均入渠コスト",
+	"avg_resource_per": "最終ノード{img}あたり:",
+	"copy_to_clipboard": "クリップボードにコピー",
+	"copied_to_clipboard": "コピーされました",
+	"avg_emptied_plane_slot": "平均艦載機スロット全滅数",
+	"avg_emptied_lbas_slot": "平均基地航空部隊全滅数",
+	
+	"node_n": "Node {0}",
+	"battle_type": "戦闘種別",
+    "battle_normal": "通常戦闘マス",
+    "battle_night_only": "開幕夜戦マス",
+    "battle_air_raid": "空襲戦マス",
+    "battle_air_only": "航空戦マス",
+	"night_battle": "夜戦する?",
+    "always_if_possible": "可能ならいつでも",
+    "only_day_rank_less_than_A": "A未満なら",
+    "only_day_rank_less_than_B": "B未満なら",
+    "only_flag_not_sunk": "敵旗艦が健在なら",
+    "resupply_cost": "資源消費タイプ",
+    "submarine_only": "潜水艦のみ",
+	"add_cost": "追加コスト(戦闘開始前に現在の残り資源から)",
+	"max": "Max", // ok
+	"player_formation": "自艦隊陣形",
+    "use_form_if_special_impossible": "特殊砲撃が不可能な場合、\u00a0{img_form}\u00a0を使用",
+	"line_ahead": "単縦陣",
+    "CF4": "第四警戒航行序列: 戦闘隊形",
+	"lbas_waves": "基地航空隊派遣",
+	"support": "支援艦隊",
+	"support_normal": "道中",
+	"support_boss": "決戦",
+	"use_smokescreen": "煙幕を使用",
+    "enable_barrage_balloon_effects": "阻塞気球を使用する",
+    "enable_atoll_node_effects": "特四式内火艇/改による越環礁攻撃を実施する",
+    "use_anchorage_repair_before_battle": "先頭前に緊急泊地修理を実施する",
+	"override_historical_bonuses": "史実ボーナス上書き",
+	"set_bonuses": "手動セット",
+	"main_fleet_set": "Main Fleet set", // ?
+	"lbas_set": "Land Base Aerial Support set", // ?
+	"friend_fleet_set": "Friend Fleet set", // ?
+	"import_bonuses_once": "まとめてインポート",
+	
+	"fleet_comp_num": "{0} - 編成 {1}",
+	"weight": "件数",
+	"import_kcnav": "KCNavから読込",
+	
+	"ship_has_customised_stats": "ステータス編集有",
+	"ship_has_unknown_stats": "不明なステータスが含まれます",
+	
+	"load_options": "編成読み込み",
+	"from_text": "デッキビルダーから:",
+    "from_text_note": "本シミュ、デッキビルダー、KC3 ReplayのJSONファイルが読み込み可能です",
+	"load": "Load", // ok
+	"fleet": "艦隊",
+	"CTF": "空母機動部隊",
+	"STF": "水上打撃部隊",
+	"TCF": "輸送護衛部隊",
+	"from_kc3_file": ".kc3 ファイルから",
+	"from_preset": "プリセットから",
+	"clear_all": "艦隊リセット",
+	"export_deckbuilder": "デッキビルダー出力",
+	"stat_level": "レベル",
+	"stat_HP": "HP",
+	"stat_true_value_unknown": "このステータスの正確な値はまだ不明です。",
+    "stat_true_value_placeholder": "このステータスの正確な値はまだ不明であり、データが不足しています。手動での設定を推奨します。",
+    "stat_true_value_db_estimate": "このステータスの正確な値はまだ不明であり、データベースに送信されたデータに基づく推定値です。",
+	"stat_Firepower": "火力",
+	"stat_Torpedo": "雷装",
+	"stat_Anti-Air": "対空",
+	"stat_Armour": "装甲",
+	"stat_Evasion": "回避",
+	"stat_Anti-Submarine": "対潜",
+	"stat_Line of Sight": "索敵",
+	"stat_luck": "運",
+	"stat_range": "射程",
+	"stat_divebomb": "爆装",
+	"stat_accuracy": "命中",
+	"stat_torpedo_accuracy": "雷撃命中",
+    // bonus_dmg, bonus_acc, and bonus_eva should have a clean line break before 'ボーナス'
+    "bonus_dmg": "ダメージボーナス",
+	"bonus_dmg_desc": "マップ全体でダメージボーナスを付与します(倍率)",
+    "bonus_acc": "命中ボーナス",
+    "bonus_acc_desc": "マップ全体で命中ボーナスを付与します(倍率)",
+    "bonus_eva": "回避ボーナス",
+    "bonus_eva_desc": "マップ全体で回避ボーナスを付与します(倍率)",
+    // Who applies what to whom and how?

```

Comment:
Value is a damage multiplier similar to "Bonus Dmg" that is applied to the player ship's attack, it is effective against the enemy flagship of the final node only and is cumulative with Bonus Dmg.
Note this is a legacy setting to remain compatible with simulations done before we figured out precise debuff effects, so it would be rarely used now.